README.rst

```diff
@@ -57,6 +57,18 @@ To check if it worked start Python and load the package:
 You're done!
 
 
+Development
+-----------
+
+First, install anaconda and run
+
+    $ conda install numpy scipy pytables
+
+After this, SAPPHiRE should work out of the box:
+
+    $ git clone ... && cd sapphire && python setup.py develop
```

Comment:
You should make it easy to copy-paste.
So add the git url to the clone command: `https://github.com/HiSPARC/sapphire.git`
And anaconda is not really needed, miniconda also suffices, you could add a link to that installer:
http://conda.pydata.org/docs/install/quick.html